frontend/src/__tests__/cypress/cypress/pages/userManagement.ts

```diff
@@ -38,12 +38,16 @@ class GroupSettingSection extends Contextual<HTMLElement> {
     return this.find().findByTestId('group-selection-error-text');
   }
 
-  findMultiGroupSelectButton() {
-    return this.find().findByRole('button', { name: 'Options menu' });
+  findMultiGroupSelectButton(admin: boolean) {
+    return this.find().findByRole('button', {
+      name: admin
+        ? 'Select the OpenShift groups that contain all Data Science administrators.'
+        : 'Select the OpenShift groups that contain all Data Science users.',
+    });
```

Comment:
This is odd.
It's the only button in the section and should use a common selector like a data-testid.
The scope of the object is already defined as admin or not, we shouldn't need to pass in an additional property to distinguish as such.